tests/bot/exts/filters/test_antimalware.py

```diff
@@ -105,20 +105,35 @@ async def test_python_file_redirect_embed_description(self):
 
     async def test_txt_file_redirect_embed_description(self):
         """A message containing a .txt file should result in the correct embed."""
-        attachment = MockAttachment(filename="python.txt")
-        self.message.attachments = [attachment]
-        self.message.channel.send = AsyncMock()
-        antimalware.TXT_EMBED_DESCRIPTION = Mock()
-        antimalware.TXT_EMBED_DESCRIPTION.format.return_value = "test"
-
-        await self.cog.on_message(self.message)
-        self.message.channel.send.assert_called_once()
-        args, kwargs = self.message.channel.send.call_args
-        embed = kwargs.pop("embed")
-        cmd_channel = self.bot.get_channel(Channels.bot_commands)
+        test_values = (
+            ("text", ".txt"),
+            ("json", ".json"),
+            ("csv", ".csv"),
+        )
 
-        self.assertEqual(embed.description, antimalware.TXT_EMBED_DESCRIPTION.format.return_value)
-        antimalware.TXT_EMBED_DESCRIPTION.format.assert_called_with(cmd_channel_mention=cmd_channel.mention)
+        for file_name, disallowed_extension in test_values:
+            with self.subTest(file_name=file_name, disallowed_extension=disallowed_extension):
+
+                attachment = MockAttachment(filename=f"{file_name}{disallowed_extension}")
+                self.message.attachments = [attachment]
+                self.message.channel.send = AsyncMock()
+                antimalware.TXT_EMBED_DESCRIPTION = Mock()
+                antimalware.TXT_EMBED_DESCRIPTION.format.return_value = "test"
+
+                await self.cog.on_message(self.message)
+                self.message.channel.send.assert_called_once()
+                args, kwargs = self.message.channel.send.call_args
+                embed = kwargs.pop("embed")
+                cmd_channel = self.bot.get_channel(Channels.bot_commands)
+
+                self.assertEqual(
+                    embed.description,
+                    antimalware.TXT_EMBED_DESCRIPTION.format.return_value
+                )
+                antimalware.TXT_EMBED_DESCRIPTION.format.assert_called_with(
+                    blocked_extension_str=disallowed_extension,
+                    cmd_channel_mention=cmd_channel.mention
+                )
 
     async def test_other_disallowed_extension_embed_description(self):
         """Test the description for a non .py/.txt disallowed extension."""
```

Comment:
Same as above, should include the other extensions in some way

Reply:
Resolved in aeaef8ff604c9ea62fdf1602200ee87f2adf7f6a.